lib/amazonka/src/Amazonka.hs

```diff
@@ -48,7 +47,7 @@ module Amazonka
     AccessKey (..),
     SecretKey (..),
     SessionToken (..),
-    Credentials (..),
+    discover,
```

Comment:
Everything's pushed people towards discover since ages. The other auth names look strange if you import them straight out of `Amazonka`, but look okay under `import qualified Amazonka.Auth as Auth` (`Auth.fromKeys _ _` etc). I will add more haddocks to remind people that they exist.